docs/openshift/kctlr-use-as3-backend.rst

```diff
@@ -88,11 +88,12 @@ Annotations
 - virtual-server.f5.com/clientssl
 - virtual-server.f5.com/serverssl
 - virtual-server.f5.com/secure-serverssl
+- virtual-server.f5.com/waf
 
 Known Issues
 ------------
 
-v1.10.0
+v1.11.0
```

Comment:
Add a new section v1.11.0 above the v1.10.0 section and add limitations specific to v1.11.0 release that are NOT part previous releases.

Reply:
Ok will make changes accordingly